source/module_elecstate/module_dm/density_matrix.cpp

```diff
@@ -458,6 +458,14 @@ template <>
 void DensityMatrix<std::complex<double>, double>::cal_DMR()
 {
     ModuleBase::TITLE("DensityMatrix", "cal_DMR");
+
+    // To check whether DMR has been initialized
```

Comment:
I don't recommand to use `WARNING_QUIT` for checking conditions which only occurred by development rather than users. 
I recommand to check them by `assert(...)` with macro definition `__DEBUG`, developers can set `-DDEBUG_INFO=ON`  during developing and testing to avoid this error.

Reply:
OK, these have been changed to `assert` with `__DEBUG` flag.